charts/gardener-dashboard/charts/runtime/templates/deployment.yaml

```diff
@@ -99,6 +99,9 @@ spec:
             - name: http
               containerPort: {{ .Values.global.containerPort }}
               protocol: TCP
+            - name: metrics
+              containerPort: {{ .Values.global.metricsContainerPort }}
+              protocol: TCP
```

Comment:
don't we also need the `prometheus.io/scrape`, `prometheus.io/port` annotation?

Reply:
the scrape job will look for our `app.kubernetes.io/name` and `.../component` as well as a port named `metrics` instead.